packages/common/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import moment from 'moment';
```

Comment:
Can we use Dayjs instead of moment? We have been trying to move that way. But I dont know if there is a constraint here.

Reply:
sure, let me try that

Reply:
dayjs requires a ¿plugin? and the implementation might be slightly different, so I'm afraid of replacing this with dayjs... 

```
var utc = require('dayjs/plugin/utc')
dayjs.extend(utc)
```

https://day.js.org/docs/en/plugin/utc

Reply:
ok, I made the change, could you quickly review and test again ?

Reply:
Interesting, I didn't know it needed a plugin for this. It looks like it's because of package size. It's *supposed* to have basically the same API, so I would expect this to work. But if it doesn't seem to, we can keep moment here. I'm not trying to add more work to this, just following the moment docs that suggest using dayjs:

https://momentjs.com/docs/#/-project-status/recommendations/

We do already have moment anyway, so its not super important to avoid it here